src/core/constants.ts

```diff
@@ -180,6 +180,10 @@ export const SWA_CONFIG_FILENAME = "staticwebapp.config.json";
 export const SWA_CONFIG_FILENAME_LEGACY = "routes.json";
 export const CUSTOM_URL_SCHEME = "swa://";
 export const OVERRIDABLE_ERROR_CODES = [400, 401, 403, 404];
+export const DEFAULT_NODE_VERSION = "16";
+export const DEFAULT_DOTNET_VERSION = "6.0";
+export const DEFAULT_DOTNET_ISOLATED_VERSION = "6.0";
+export const DEFAULT_PYTHON_VERSION = "3.8";
```

Comment:
How did we decide what the default version should be? I am afraid we will have to keep this list updated if we hardcode the default versions.

Reply:
We're getting this info from a file in StaticWebApps repository. Will figure out a way to keep this in a common place and use it later.